src/postman/__tests__/util.test.ts

```diff
@@ -32,7 +32,7 @@ describe('traverseItemsAndGroups()', () => {
 
       traverseItemsAndGroups(new ItemGroup({ item: [{ item: [{}] }] }), mockItemCallback, mockGroupItemCallback);
 
-      expect(mockItemCallback).toHaveBeenCalledTimes(2);
+      expect(mockItemCallback).toHaveBeenCalledTimes(1);
```

Comment:
Why is this being modified?

Reply:
Well, I'm fixing the bug I placed there before.

Basically I assumed (and somehow wrongly verified) that `ItemGroup.forEachItem(Group)?()` does not traverse the tree recursively. I was wrong, it does. So our traverse implementation is basically useless and can be removed.

In order to make things even more complicated for the reviewer, I removed traversing in following "refactor" PR https://github.com/stoplightio/http-spec/pull/69/files#diff-4686c87b4e40c955db91ee7283af86b2L5